src/cca/scripts/cordova_core_connector.ts

```diff
@@ -17,7 +17,7 @@ export interface OnEmitModuleFactory extends
   freedom.FreedomModuleFactoryManager<OnEmitModule> {};
 
 console.log('Loading core');
-export var uProxyAppChannel = freedom(
+export var uProxyAppChannelPromise = freedom(
```

Comment:
It doesn't look like this is used outside of this source file anymore in which case this should not be `export`-ed.